test/e2e/install_installer_test.go

```diff
@@ -36,24 +36,6 @@ func TestInstallUpdaterSuite(t *testing.T) {
 	})
 }
 
-func (s *installUpdaterTestSuite) TestInstallUpdater() {
-	t := s.T()
-	vm := s.Env().RemoteHost
-	cmd := fmt.Sprintf("DD_INSTALLER=true DD_API_KEY=%s DD_SITE=\"datadoghq.com\" bash -c \"$(cat scripts/install_script_agent7.sh)\"", apiKey)
-	output := vm.MustExecute(cmd)
-	t.Log(output)
-	defer s.purge()
-
-	s.assertInstallScript(true)
-	s.assertInstallerInstalled()
```

Comment:
I believe `assertInstallerInstalled` is solely used in this removed function, could we remove it too ? Same for `assertUninstallInstaller`, `uninstallInstaller`

Reply:
Ah yes good catch, thanks